server/app/components/product-header.tsx

```diff
@@ -0,0 +1,32 @@
+import { Card } from "@tremor/react";
+
+interface ProductHeaderProps {
+  product: {
+    image_url: string;
+    name: string;
+  };
+}
+
+const ProductHeader = ({ product }: ProductHeaderProps) => (
+  <>
+    {product?.image_url && (
+      <Card className="mx-auto grid w-full grid-cols-1 gap-1 space-y-4 md:h-full md:grid-cols-7 md:gap-6 lg:w-3/4">
+        <span className="grid justify-center">
+          <img
+            src={product.image_url.replace("US120", "US240")}
+            alt="Product"
+            width={120}
+          />
+        </span>
+
+        <span className="md:col-span-6">
+          <h1 className="text-center text-xl font-semibold md:text-left">
+            {product?.name}
+          </h1>
```

Comment:
`gap` and `space` do the same thing. I'm assuming you were trying to use it to center the text?
```suggestion
      <Card className="mx-auto grid w-full grid-cols-1 gap-1 md:h-full md:grid-cols-7 md:gap-6 lg:w-3/4">
        <span className="grid justify-center">
          <img
            src={product.image_url.replace("US120", "US240")}
            alt="Product"
            width={120}
          />
        </span>

        <span className="md:col-span-6">
          <h1 className="my-auto text-center text-xl font-semibold md:text-left">
            {product?.name}
          </h1>
```

Reply:
Text gets misaligned with this

Reply:
Did you remove `space-y-4` as well?

Reply:
![image](https://github.com/6monthslater/6monthslater/assets/12688112/351e1bf0-62df-4732-8afd-2a11dd989917)

yea

Reply:
Try 
```
        <span className="grid justify-center md:col-span-6">
          <h1 className="my-auto text-center text-xl font-semibold md:text-left">
          ```